plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/androidx/room/visitor/RoomCallMethodVisitor.kt

```diff
@@ -62,19 +62,23 @@ class RoomCallMethodVisitor(
         when {
             (opcode == Opcodes.INVOKEVIRTUAL && name == SET_TRANSACTION_SUCCESSFUL) -> {
                 // the original method wants to return, but we intervene here to set status
+                instrumenting.set(true)
```

Comment:
would make sense to put this before and after `when` instead of each case?

Reply:
Will be gone in the follow-up PR, sorry for the mess